File: 241120/짝수이면서 합이 5의 배수인 수/an-even-number-with-a-multiple-of-5-in-the-sum.java
import java.util.Scanner;
public class Main {

    public static boolean snum(int n){
        if(n % 2 == 0){
            if((n /10 + n % 10 ) % 5 == 0){
                return true;
            }
        }
        return false;
    }
    public static void main(String[] args) {
        // 여기에 코드를 작성해주세요.
        Scanner sc = new Scanner(System.in);
        int n = sc.nextInt();

        if(snum(n)){
            System.out.print("Yes");
        }else{
            System.out.print("No");
        }
    }
}

File: 241120/짝수이면서 합이 5의 배수인 수/an-even-number-with-a-multiple-of-5-in-the-sum.py
n = int(input())

def snum(num):
    if num % 2 == 0:
        if (num % 10 + num // 10) % 5 == 0:
            return True 
    return False 

if snum(n):
    print("Yes")
else:
    print("No")